db/migrate/20150422060517_create_course_condition_achievements.rb

```diff
@@ -1,7 +1,8 @@
-class CreateCourseConditionsAchievements < ActiveRecord::Migration
+class CreateCourseConditionAchievements < ActiveRecord::Migration
   def change
     create_table :course_condition_achievements do |t|
       t.references :achievement, foreign_key: { references: :course_achievements }
+      t.timestamps null: false
```

Comment:
Actually there's no need to add a timestamp here because the condition table already has a timestamp column. I'll will just revert the changes